### PR TITLE
YMMV

### PR DESCRIPTION
**Pull request by danielwestendorf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,35 +57,54 @@ The CTO, first engineer, cofounder, or anyone at a pre-seed startup who is accou
...
   - [ ] To make a decision, look for answers in analytics and metrics
   - [ ] Reporting: Sooner or later, the data you can query from the database will need to be turned into reporting features for your customers
-
+  - [ ] Customer suggested solutions are usually wrong. Dig into the "why" before weighing the "how"; the need is usually legitimate
+  - [ ] Never implement a feature just for one customer; it will hurt you more in the future than help you in the short term
+    - Exception of paid/sponsored features, which should take maintenance and support into consideration
+  - [ ] Ensure existing customers want a feature before you start building it
+    - Potential customers always want the moon; rarely will just-one-more-feature! start landing customers. Choose features many leads want, not just a few
+  - [ ] Have a vision for your product and work towards it. Passing on customers because of features that don't fit your vision will be painful, but will pay off in the long run.
+  - [ ] Users are stupid. Dumb down features/UIs until a flea could use them.
```
Comment (stockandawe) [inline README.md]: @danielwestendorf , can we update this to less strong language?
Suggestion:
- [ ] Aim for a "simple" product. Simple is not easy, but it's what your customers and users need. The craft of product is doing the hard work of figuring out the complexity to make using it simple for your customers.

File: README.md [hunk 2]
```diff
@@ -84,4 +114,8 @@ The CTO, first engineer, cofounder, or anyone at a pre-seed startup who is accou
   - [ ] Find a way to mentor someone (within or outside your company)
   - [ ] Join a community of CTOs, engineers, technical leaders who you can knowledge share with
   - [ ] In your day-to-day, balance big projects and small enhancements/bug fixes 
+  - [ ] Group email address for ownership of all external services
```
Comment (stockandawe) [inline README.md:126]: This is such a good one. Nice suggestion!

**Review by stockandawe — CHANGES_REQUESTED** [review]
Thanks @danielwestendorf. Awesome suggestions and thanks for making this better.
Just one suggestion.

File: README.md (hunk 1 above)
Comment (danielwestendorf) [reply README.md]:
```suggestion
  - [ ] Aim for a "simple" product. Simple is not easy, but it's what your customers and users need. The craft of product is doing the hard work of figuring out the complexity to make using it simple for your customers.
```

**Pull request merged**